src/modules/user/service.js

```diff
@@ -18,12 +23,17 @@ const createUser = async (data) => {
       email,
       avatarUrl,
       password: passwordHash,
+      phoneNumber,
     });
 
     return user;
   } catch (err) {
     if (err instanceof db.Sequelize.UniqueConstraintError) {
-      throw new ValidationError(EMAIL_IS_NOT_UNIQUE_ERROR_MESSAGE);
+      if (err.fields['users.phone_number']) {
```

Comment:
I would recommend to add new error class for `409 Conflict` type
Figure out with insides of `err` object and decide which actually fields can be dynamically passed to Conflict instance error
Then it should be easy to use
So instead of doing two `if's` I would prefer to see something like this:
```
if (err instanceof db.Sequelize.UniqueConstraintError) {
   throw new ConflictError(err[please decide how it can be used to determine it's email or phone or something other])
}
```